client.go

```diff
@@ -1388,14 +1380,18 @@ func (cli *Client) downloadContext(ctx context.Context, mxcURL id.ContentURI) (*
 		return nil, err
 	}
 	req.Header.Set("User-Agent", cli.UserAgent+" (media downloader)")
-	cli.LogRequest(req)
+	cli.RequestStart(req)
+	downloadStart := time.Now()
 	if resp, err := cli.Client.Do(req); err != nil {
+		cli.LogRequestDone(req, resp, err, 0, time.Since(downloadStart))
```

Comment:
This call is wrong since `resp` is nil, although now I wonder if there should be some common place for logging failed requests 🤔